### ISSUE TITLE
gutil.colors.stripColor is not a function

### ISSUE THREAD

**Issue by jakemmarsh** [dlmanning/gulp-sass#458]
`gutil.colors` is an instance of [Chalk](https://github.com/chalk/chalk), which recently removed the `stripColor` method [here](https://github.com/chalk/chalk/commit/04cae226cc0fc11fd7898f3fa91fdc4a3b3e496b).

I'm now getting this error any time a SASS error is encountered. Stack trace:

```
TypeError: gutil.colors.stripColor is not a function
    at errorM (/Users/jake/Repos/monolist-web/node_modules/gulp-sass/index.js:129:36)
    at Object.callback (/Users/jake/Repos/monolist-web/node_modules/gulp-sass/index.js:144:18)
    at options.error (/Users/jake/Repos/monolist-web/node_modules/gulp-sass/node_modules/node-sass/lib/index.js:280:32)
```

**Comment by scott-goetz** [comment]
I'm having the same issue.

**Comment by jakemmarsh** [comment]
I've put out a simple fix in #459

**Comment by xzyfer** [comment]
This looks like a temporary blip in a bad release. The bad release has since been removed from npm.
More information in https://github.com/chalk/chalk/issues/106

**Comment by C-Bouthoorn** [comment]
This issue is now re-occuring again on gulp-sass, with the release of Chalk 2.0 a week or two ago. In https://github.com/chalk/chalk/releases/tag/v2.0.0 they tell us to use `strip-ansi` as used by this temporary fix. Could you please merge this fix?

**Comment by C-Bouthoorn** [comment]
Bump

**Comment by Lemmmy** [comment]
Having issues here too.

**Comment by C-Bouthoorn** [comment]
@Lemmmy I made a temporary dirty workaround with:

```coffeescript
# CoffeeScript
## -------------------------------
##   Dirty fix for gulp-sass#458
## -------------------------------
require 'gulp-util'

require.cache[
    Object.keys(require.cache)
        .find((x) -> x.indexOf('gulp-util/index') >= 0)
].exports.colors.stripColor = require 'strip-ansi'
## -------------------------------
```

or

```javascript
// JavaScript
// -------------------------------
//   Dirty fix for gulp-sass#458
// -------------------------------

require('gulp-util')

require.cache[
    Object.keys(require.cache)
        .find((x) => x.indexOf('gulp-util/index') >= 0)
].exports.colors.stripColor = require('strip-ansi')
// -------------------------------
```

but I'd like to see it fixed here
